shared/ghutil/pullrequest.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// pullrequest.go provides generic functions related to PullRequest
+
+package ghutil
+
+import (
+	"fmt"
+
+	"github.com/google/go-github/github"
+)
+
+const (
+	// PullRequestOpenState is the state of open PullRequest
+	PullRequestOpenState PullRequestStateEnum = "open"
+	// PullRequestCloseState is the state of closed PullRequest
+	PullRequestCloseState PullRequestStateEnum = "closed"
+	// PullRequestAllState is the state for all, useful when querying PullRequest
+	PullRequestAllState PullRequestStateEnum = "all"
```

Comment:
this seems weird as a pr state. What does it mean to have pr with all state? Maybe replace with any?

Reply:
keep this all if GH expects all though.

Reply:
yep, GH only accepts `open`, `closed`, or `all`